workflows/create-pull-request.yaml

```diff
@@ -0,0 +1,59 @@
+name: Create Pull request
+
+on:
+  schedule:
+    # Runs "at minute 44 past midnight" (see https://crontab.guru)
+    - cron: "44 0 * * *"
+
+jobs:
+  createPullRequest:
+    runs-on: ubuntu-latest
+    env:
+      bot_username: BotName
+      bot_email: BotMail@gmail.com
+    strategy:
+      fail-fast: false
+      matrix:
+        repos: ["de-german", "fr-french"]
+    steps:
+      - name: Fetch translation repository
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.PAT }}
+          repository: solidity-docs-test/${{ matrix.repos }}
+          fetch-depth: "0"
+
+      - name: Fetch this repository
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.PAT }}
+          repository: solidity-docs/translation-guide
+          path: .github-workflow
+
+      - name: Create merge commit
+        id: merge
+        run: |
+          ./.github-workflow/scripts/merge-conflicts.sh
+          ./.github-workflow/scripts/generate-PR-body.sh
+
+      - name: Remove this repository
+        run: |
+          rm -rf .github-workflow
+
+      - name: Create Pull Request
+        id: cpr
+        uses: peter-evans/create-pull-request@v3
+        with:
+          token: "${{ secrets.PAT }}"
+          commit-message: "Update report"
+          committer: "${{ env.bot_username }} <${{ env.bot_email }}>"
+          author: "${{ env.bot_username }} <${{ env.bot_email }}>"
+          branch: "${{ steps.merge.outputs.branchName }}"
+          title: "${{ env.pr_title }}"
+          body: "${{ env.pr_body }}"
+          labels: |
+            report
+            automated pr
+          assignees: "${{ env.bot_username }}"
+          reviewers: o-lenczyk
+          team-reviewers: o-lenczyk
```

Comment:
I'd create variables for all these in the top-level `env` block as well since this is something we'll want to customize.
Also, not sure if it makes sense to assign the PR to the bot :)

Reply:
Actually, this is something that will differ per repo. Maybe we should make it configurable by translators? I.e. each repo could contain a small YAML or JSON file where these are listed and the bot could load it and use values from it.

If this was a reusable action invoked from a workflow in each repo, these could also be parameters.

Reply:
there is already `lang` folder in central repository

Reply:
Great idea. Yeah. we should be taking these from there.

I think it should also be fine to add any extra fields you need to these JSON files if you need that.

Reply:
What I did, for now, is to take a random maintainer as an assignee, and another random maintainer as a reviewer. Do you think it can work?

Reply:
I suspect that translators might prefer to be able to set a list of reviewers/assignees by themselves. But it's just the default value so not like it's hard to change. I'm fine with doing it either way, choose the one you think is best.

Reply:
I think it is best to leave it as it is now. We can change it when the lang.json file would have more information.